third_party/blink/renderer/platform/scheduler/main_thread/page_scheduler_impl.cc

```diff
@@ -206,6 +207,11 @@ PageSchedulerImpl::~PageSchedulerImpl() {
   main_thread_scheduler_->RemovePageScheduler(this);
 }
 
+void
+PageSchedulerImpl::BreakLinkages() {
+  delegate_ = nullptr;
```

Comment:
Hum... once this is unset, `IsOrdinary` will return `true`. I wonder if this will cause more issues down the line?

Reply:
that's a great question.  I could store it off in BreakLinkages and return it from IsOrdinary.  It doesn't appear to cause issues, though.